### PR TITLE
issue-3-user-can-favorite-park

### PR DESCRIPTION
**Pull request by brendatrieu**
https://user-images.githubusercontent.com/121825299/222628228-c03d34c2-9178-4ce4-9a0a-92d25d29c64f.mov

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,6 +78,15 @@ <h4 class="subheading">Activities: </h4>
           </div>
         </div>
       </section>
+      <section class="" id="Favorites">
```
Comment (shawnkost) [inline index.html:81]: If you don't plan to give this a `class`, remove the `class` attribute

File: js/refData.js [hunk 2]
```diff
@@ -76,3 +76,11 @@
...
 //   }
 // };
+// var targetIndex = 0;
+// event.target.className = 'fa-regular fa-star';
+// if (data.view === 'individual-park') {
+//   targetIndex = data.favorites.indexOf(data.targetPark);
+//   return data.favorites.splice(targetIndex, 1);
+// }
+// targetIndex = data.favorites.indexOf(event.target.closest('.park-high-lvl').getAttribute('id'));
+// return data.favorites.splice(targetIndex, 1);
```
Comment (shawnkost) [inline js/refData.js]: Remove old code

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just a couple small changes!

File: index.html (hunk 1 above)
Comment (brendatrieu) [reply index.html:81]: Class will toggle on and off between views.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**